mitodl/open-discussions
# Default image for courses/bootcamps

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2141 

#### What's this PR do?
Displays a default image for courses & programs that don't have one of their own.

#### How should this be manually tested?
- Import edx data if you haven't done so (`docker-compose run web python manage.py backpopulate_edx_data`)
- Go to `/courses/search`, and search for "Harry's Lessons on Life"
- The search result card for that course should (try to) display the default course image via embedly - check the URL, it should be `https://i.embed.ly/1/display/crop/?key=<key>&url=http%3A%2F%2Fod.odl.local%3A8063%2Fstatic%2Fimages%2Fdefault_resource_thumb.jpg&height=130&width=306&grow=true&errorurl=http://od.odl.local:8063/static/images/blank.png`
- If you want to verify that it will actually display, you can use [ngrok](https://ngrok.com/) so embedly can reach the URL.
- Click the card to bring up the course drawer, it should also try to display the default image.


#### Screenshots
<img width="846" alt="Screen Shot 2019-09-16 at 9 55 14 AM" src="https://user-images.githubusercontent.com/187676/64964024-3c37d080-d868-11e9-8bcb-f0461f98596b.png">

<img width="502" alt="Screen Shot 2019-09-16 at 9 55 21 AM" src="https://user-images.githubusercontent.com/187676/64964044-435ede80-d868-11e9-8e60-2832a98fdd94.png">
